common/test/acceptance/tests/studio/test_studio_html_editor.py

```diff
@@ -3,12 +3,17 @@
 """
 from __future__ import absolute_import
 
+import os
+
 from common.test.acceptance.fixtures.course import XBlockFixtureDesc
 from common.test.acceptance.pages.studio.container import ContainerPage, XBlockWrapper
 from common.test.acceptance.pages.studio.html_component_editor import HTMLEditorIframe, HtmlXBlockEditorView
 from common.test.acceptance.pages.studio.utils import add_component, type_in_codemirror
 from common.test.acceptance.tests.studio.base_studio_test import ContainerBase
 
+UPLOAD_SUFFIX = 'data/uploads/studio-uploads/'
+UPLOAD_FILE_DIR = os.path.abspath(os.path.join(__file__, '../../../../', UPLOAD_SUFFIX))
```

Comment:
seems like a patchy way to do this.

Reply:
yeah, the other way is `UPLOAD_FILE_DIR = Path(__file__).abspath().dirname().dirname().dirname().dirname() + UPLOAD_SUFFIX`

Please suggest

Reply:
I was more skeptical about the path. Don't know why it is that complicated.

Reply:
:-) ok.